include/boost/geometry/formulas/vertex_latitude.hpp

```diff
@@ -0,0 +1,149 @@
+// Boost.Geometry
+
+// Copyright (c) 2016 Oracle and/or its affiliates.
+
+// Contributed and/or modified by Vissarion Fysikopoulos, on behalf of Oracle
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_GEOMETRY_FORMULAS_MAXIMUM_LATITUDE_HPP
+#define BOOST_GEOMETRY_FORMULAS_MAXIMUM_LATITUDE_HPP
+
+#include <boost/geometry/formulas/spherical.hpp>
+
+namespace boost { namespace geometry { namespace formula
+{
+
+/*!
+\brief Algorithm to compute the vertex latitude of a geodesic segment. Vertex is
+a point on the geodesic that maximizes (or minimizes) the latitude.
+\author See
+    Wood - Vertex Latitudes on Ellipsoid Geodesics, SIAM Rev., 38(4), 637–644,
+    1996
+*/
+
+template <typename CT>
+class vertex_latitude
+{
+
+public:
+
+    struct vertex_lat_result
+    {
+        vertex_lat_result()
+            :   north(0),
+              south(0)
+        {}
+
+        CT north, south;
+    };
+
+    template <
+            typename T1,
+            typename T2
+            >
+    static inline vertex_lat_result spherical(T1 const& lon1,
```

Comment:
Also minor, if it fits in one line (80 chars), you may write it on one line (template <typename T1, typename T2>
If it does not fit on one line, it should be indented like functions do, so < goes on next line, aligned under template